libraries/rush-lib/src/logic/operations/OperationExecutionManager.ts

```diff
@@ -86,14 +89,16 @@ export class OperationExecutionManager {
       beforeExecuteOperationAsync: beforeExecuteOperation,
       afterExecuteOperationAsync: afterExecuteOperation,
       onOperationStatusChangedAsync: onOperationStatusChanged,
-      beforeExecuteOperationsAsync: beforeExecuteOperations
+      beforeExecuteOperationsAsync: beforeExecuteOperations,
+      cobuildConfiguration
     } = options;
     this._completedOperations = 0;
     this._quietMode = quietMode;
     this._hasAnyFailures = false;
     this._hasAnyNonAllowedWarnings = false;
     this._changedProjectsOnly = changedProjectsOnly;
     this._parallelism = parallelism;
+    this._cobuildConfiguration = cobuildConfiguration;
```

Comment:
I don't like this. Cobuilds are supposed to be confined to the `CacheableOperationPlugin`, not finding their way into `OperationExecutionManager`. How can we generalize the data we need here so that it isn't specific to the concept of cobuilds?